pyroSAR/drivers.py

```diff
@@ -2400,7 +2443,7 @@ def __exit__(self, exc_type, exc_val, exc_tb):
     
     def drop_element(self, scene, table):
```

Comment:
This method is a bit problematic. I think it is safer to remove the `table` argument and just say a scene is removed from BOTH data and duplicates. There is no reason to give the option to a user to just remove a scene from one or the other.  
IF this is needed somewhere in other methods it is safer to do it in a separate hidden method.
 Removing elements from other custom tables might be relevant for the future, but for now please remove the `TODO`.  
In summary, let's keep this method limited to functionality that is relevant and safe to use for a user.
I know, I had already accepted this method in some other PR, but I just realized this. In the end the method could look like this:
```Python
def drop_element(self, scene):
    """
    Drop an entry from the database.

    Parameters
    ----------
    scene: ID
        a SAR scene
    Returns
    -------
    """
    delete_statement = self.data_schema.delete().where(self.data_schema.c.scene == scene)
    self.conn.execute(delete_statement)
    
    delete_statement = self.duplicates_schema.delete().where(self.duplicates_schema.c.scene == scene)
    self.conn.execute(delete_statement)

    print('Entry with id {} was dropped!'.format(scene))
```
...just a quick suggestion.